clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -283,6 +283,9 @@ public FileStatus getFileStatus(Path path) throws IOException {
             Path filePath = path.makeQualified(this.uri, this.workingDirectory);
             return new FileStatus(length, false, 0, 0, modificationTime, filePath);
         } catch (ApiException e) {
+            if (e.getCode() == 404) {
```

Comment:
```suggestion
            if (e.getCode() == HttpStatus.SC_NOT_FOUND) {
```

Reply:
Gotta love that "`SC_`" prefix on a (scoped!) enum values.   Brings out the worst features of Java and C...